### PR TITLE
Centos 8 update

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

**Comment by ramereth** [comment]
@TheREK3R please rebase and resolve conflicts.

File: test/cookbooks/osl-git-test/metadata.rb [hunk 1]
```diff
@@ -10,3 +10,4 @@
 depends          'osl-git'
 
 supports         'centos', '~> 7.0'
+supports         'centos', '~> 8.0'
```
Comment (ramereth) [inline test/cookbooks/osl-git-test/metadata.rb:13]: There's no need for this in this metadata.rb. Please go ahead and remove all of these.

File: test/cookbooks/osl-git-test/metadata.rb [hunk 2]
```diff
@@ -7,6 +7,4 @@
 description      'Installs/Configures osl-git-test'
 version          '0.1.0'
 
-depends          'osl-git'
-
-supports         'centos', '~> 7.0'
+depends          'osl-git'
```
Comment (ramereth) [inline test/cookbooks/osl-git-test/metadata.rb:10]: Please fix the newline on this file.

Comment (ramereth) [reply test/cookbooks/osl-git-test/metadata.rb:9]: No I meant this line is actually missing a newline (see the red icon). It should have one here.

Comment (TheREK3R) [reply test/cookbooks/osl-git-test/metadata.rb]: sorry, updated

**Comment by TheREK3R** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.3.1.  Have a nice day!

**Pull request merged**